pump-jack-motion-monitoring/README.md

```diff
@@ -0,0 +1,132 @@
+
+```json
+{"req":"env.default","name":"pump_jack_notecard_configuration_version","text":"1.0.0"}
+```
+
+Next is `hub.set`, and you'll want to edit this request to include your Product UID in a `product` key-value pair, and optionally a serial number with the `sn` key and value. `voutbound` and `vinbound` are [voltage-variable values](https://dev.blues.io/notecard/notecard-walkthrough/low-power-design/#modem-power-management) that you can use to configure the sync behavior of the Notecard based on the state of its battery. `voutbound` corresponds to periodic syncs from the Notecard to Notehub, while `vinbound` corresponds to syncs from Notehub to Notecard.
+
+```json
+{"req":"hub.set","mode":"periodic","align":true,"voutbound":"usb:10;high:360;normal:720;low:1440;dead:0","vinbound":"usb:10;high:180; normal:180;low:3 60;dead:0"}
+```
+
+The `card.voltage` request [`mode` key](https://dev.blues.io/reference/notecard-api/card-requests/#card-voltage) is used set voltage thresholds based on how the Notecard will be powered, and which impacts the voltage-variable settings above.
+
+```json
+{"req":"card.voltage","mode":"lipo"}
+```
+
+Next, we'll configure the motion detection and tracking aspects of the app, starting with `card.motion.mode`. [This request](https://dev.blues.io/reference/notecard-api/card-requests/#card-motion-mode) is used to set-up some of the motion monitoring parameters that will be used with `card.motion.track` below. `"start":true` turns on the accelerometer and tracking, `sensitivity` is used to tune the accelerometer sample rate, and `seconds` is used to specify the period to use for bucketing movements together for automated motion tracking. More on that in a minute.
+
+```json
+{"req":"card.motion.mode","start":true,"sensitivity": -1,"seconds":60}
+```
+
+Since we're not using a host, we need to configure the Notecard to automatically capture motion data with [`card.motion.track`](https://dev.blues.io/reference/notecard-api/card-requests/#card-motion-track). `"start":true` starts automated motion tracking, which will send a motion Note in the default `_motion.qo` Notefile at least every 20 minutes (`"minutes":20`) and by examining the last 20 buckets (`"count":20`) of 60 seconds each (as set in `card.motion.mode` above). The `threshold` key sets the number of buckets that need to have at least one movement in order for a tracking note to be added.
+
+```json
+{"req":"card.motion.track","start":true,"minutes":20,"count":20,"threshold":1}
+```
+
+The `card.motion.sync` request is used to automatically sync motion-triggered Notefiles on an interval specified with the `minutes` key. Here, we're setting the sync interval to once an hour and using `"start":true` to kick off automated syncs.
+
+```json
+{"req":"card.motion.sync","start":true,"minutes":60}
```

Comment:
Should this be `{"req":"card.motion.sync","start":true,"minutes":60,"count":60,"threshold":1}`  ?
The source code here seems to suggest it should be:
https://github.com/blues/notecard/blob/7ddff5fdc0aa2882b7db7b5e06c81753f5849960/src/app/accel.c#L930
I think the intent is to sync if there has been any motion in the last 60 minutes ?